lib/metro-transit.js

```diff
@@ -2,6 +2,7 @@ var _ = require('lodash');
 
 var props = require('../resources/properties');
 var stations = require('../resources/stations');
+var abbreviations = require('../resources/abbreviations');
```

Comment:
I'm not a fan of how you are using this in multiple places. Can we build the functions you want inside UTILS to expose the individual lists you want or something?